primer.md

```diff
@@ -102,10 +102,13 @@ either the event producer or event consumer.
 
 CloudEvents, at its core, defines a set of metadata, called attributes, about
 the event being transferred between systems, and how those pieces of metadata
-should appear in that message. The metadata is not meant to duplicate any of
-the application data of the event itself, rather it is just the minimal
-information needed to route the request to the proper component that will
-process the event.
+should appear in that message. This metadata is meant to be the minimal 
+set of information needed to route the request to the proper component that
+will process the event. So, while this might mean that some of the application
+data of the event itself might be duplicated as part of the CloudEvent's set
+of properties, this is to be done solely for the purpose of proper delivery
+of the message.  Data that is meant strictly for use by the component
```

Comment:
I don't think we should restrict the metadata purpose to be solely for routing/delivery. Some of existing CloudEvents metadata are not for routing/delivery purpose.

Reply:
Suggest to add the following, which I presented in today's WG meeting, to avoid misunderstanding that only "key-value" pair format is allowed: 
Metadata can be in the format of “key-value” pair or in the format of “classification/property bag” at the root level in both the spec.md and the extension spec.

Reply:
I modified the text to address your first comment. The 2nd comment is better addressed down below in the "extensions" section and I tried to add some text in there - see what you think.

Reply:
@duglin My point is that we should not restrict the format of metadata in main spec.md either. Metadata can be in the format of “key-value” pair or in the format of “classification/property bag” at the root level in both the spec.md and the extension spec.

Reply:
This feels like its leaving the boundary of this PR. If you'd like to add some text to the "Type System" part of the spec to make it clear that the types listed there are just the ones we happen to use today and that we could add more later, then please open up a new PR and suggest some text.